e2e/fixtures/ssr-basic/src/main.tsx

```diff
@@ -10,7 +10,7 @@ const rootElement = (
   </StrictMode>
 );
 
-if (import.meta.env.WAKU_HYDRATE) {
+if (document.body.dataset.hydrate) {
```

Comment:
I know this isn't optimal.
We may revisit this in the future, or maybe it's okay with the Waku target.